src/app/projects/new-project/new-project.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { ProjectsService } from '../projects.service';
+
+@Component({
+  selector: 'app-new-project',
+  templateUrl: './new-project.component.html',
+  styleUrls: [ './new-project.component.css' ]
+})
+export class NewProjectComponent implements OnInit {
+
+  id: number;
+  projectName: string;
+  description: string;
+  coefficient = 1.55;
```

Comment:
możesz tutaj stworzyć cały obiekt projektu i w nim sobie trzymać to co user wpisze